product_warranty/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Product Warranty",
+    "application": True,
+    "depends": ["product", "account", "stock", "sale_management"],
```

Comment:
why stock is needed over here ? :thinking: 
```suggestion
    "depends": ["account", "stock", "sale_management"],
```

account already depends on product